pulsar-function-go/conf/conf.go

```diff
@@ -77,6 +77,9 @@ type Conf struct {
 	UserConfig                  string `json:"userConfig" yaml:"userConfig"`
 	//metrics config
 	MetricsPort int `json:"metricsPort" yaml:"metricsPort"`
+
+	ClientAuthPlugin string `json:"clientAuthPlugin" yaml:"clientAuthPlugin"`
+	ClientAuthParams string `json:"clientAuthParams" yaml:"clientAuthParams"`
```

Comment:
Nit: is there a reason this uses `Auth` instead of `Authentication` for these field names? I think it is valuable to have identical field names across languages, when possible, to make it easy to transition between clients. That won't work for python, which uses snake casing, but I still think it could be helpful here. What is your perspective @liangyuanpeng?

Reply:
It's kind of a convention across different language runtimes:
https://github.com/apache/pulsar/search?q=ClientAuthPlugin&type=

Reply:
The convention for functions is to use `Authentication`:

https://github.com/apache/pulsar/blob/82237d3684fe506bcb6426b3b23f413422e6e4fb/pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/AuthenticationConfig.java#L30-L31

Reply:
The convention diverged.

I'm fine with either. Let's conform to one.